server/lib/activity-event.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (event, data, request, queryParams) {
+  data.op = 'activityEvent';
+  data.event = event;
+  data.userAgent = request.headers['user-agent'];
+
+  if (queryParams) {
+    queryParams.forEach(function (param) {
```

Comment:
Huh, I'm surprised we don't use underscore or lodash on the server side. You'd be able to use `_.pick`.